src/components/svgicon/SVGIcon.stories.tsx

```diff
@@ -982,6 +982,14 @@ export const Status: StoryObj = {
           </Text>
         </FlexContainer>
       </CopyToClipboard>
+      <CopyToClipboard text="successManualFilled">
+        <FlexContainer direction="column">
+          <SVGIcon name="successManualFilled" />
+          <Text size="xs" mt=".25rem">
+            successManualFilled
+          </Text>
+        </FlexContainer>
+      </CopyToClipboard>
```

Comment:
Please Add `statusSuccessDetailed` to story book too as it is missing. 
It would be also nice to update colour of status icons. It is hard to preview them now: (

As in example, I've applied green to one of them
<img width="1445" alt="Screenshot 2024-12-04 at 13 03 13" src="https://github.com/user-attachments/assets/ef47cfcd-a1da-4fdb-b589-87208b4a7d67">